docs/Prelude.md

```diff
@@ -983,4 +983,20 @@ instance showArray :: (Show a) => Show (Array a)
 instance showOrdering :: Show Ordering
 ```
 
+#### `NaturalTransformation`
+
+``` purescript
+type NaturalTransformation f g = forall a. f a -> g a
+```
+
+A natural transformation
+
+#### `Natural`
+
+``` purescript
+type Natural f g = NaturalTransformation f g
```

Comment:
Hah I argued for the exact opposite in https://github.com/purescript/purescript-free/pull/37 :smile:

One minor problem, if we don't bump the version as a breaking change bump after merging this it will probably break a bunch of stuff. Halogen for sure.